bitwarden_access_manager.py

```diff
@@ -0,0 +1,672 @@
+                result = (
+                    result + "member_access = [\n" + ",\n".join(member_access_lines) + ",\n]\n"
+                )
+            else:
+                result = result + "member_access = []\n"
+
+        if self.group_access is not None:
+            group_access_lines = ["  " + a.format_toml() for a in sorted(self.group_access)]
+            if len(group_access_lines) > 0:
+                result = (
+                    result + "group_access = [\n" + ",\n".join(group_access_lines) + ",\n]"
+                )
+            else:
+                result = result + "group_access = []"
+        return result
+
+class BitwardenClient(NamedTuple):
+    connection: HTTPSConnection
+    bearer_token: str
+
+    @staticmethod
+    def new(client_id: str, client_secret: str) -> BitwardenClient:
+        response = requests.post(
+            "https://identity.bitwarden.com/connect/token",
+            headers={ "Content-Type": "application/x-www-form-urlencoded" },
+            data={
+                "grant_type": "client_credentials",
+                "scope": "api.organization",
+                "Accept": "application/json",
+            },
+            auth=(client_id, client_secret),
+        )
+        bearer_token = response.json()["access_token"]
+        return BitwardenClient(HTTPSConnection("api.bitwarden.com"), bearer_token)
+
+    def _http_get(self, url: str) -> HTTPResponse:
+        self.connection.request(
+            method="GET",
+            url=url,
+            headers={
+                "Accept": "application/json",
+                "Authorization": f"Bearer {self.bearer_token}",
+            },
+        )
+
+        return self.connection.getresponse()
+
+    def get_group(self, id: str) -> Any:
+        return json.load(self._http_get(f"/public/groups/{id}"))
+
+    def get_groups(self) -> Iterable[Group]:
+        data = self._http_get(f"/public/groups")
+        groups = json.load(data)
+        for group in groups["data"]:
+            yield Group(
+                id=group["id"],
+                name=group["name"],
+                access_all=group["accessAll"],
+            )
+
+    def get_collection_members(self, groups: Any) -> Iterable[MemberCollectionAccess]:
+            for group in groups:
+                group_id = group["id"]
+
+                data = self._http_get(f"/public/groups/{group_id}/member-ids")
+                member_ids = json.load(data)
+
+                for member_id in member_ids:
+                    data = self._http_get(f"/public/members/{member_id}")
+                    member = json.load(data)
+                    yield MemberCollectionAccess(
+                        name=member["name"],
+                    )
+
+    def get_collection_groups(self, groups: Any) -> Iterable[GroupCollectionAccess]:
+
+            for group in groups:
+                if group["readOnly"] == True:
+                    access = "readonly"
```

Comment:
```suggestion
                    access = Access.READONLY
```
If we would use an enum, then here we'd use the enum variant directly. (Also below.)